lib/cisco_node_utils/interface.rb

```diff
@@ -1019,9 +1019,19 @@ def default_switchport_mode_private_vlan_host_association
     end
 
     def prepare_array(is_list)
```

Comment:
Absolutely agree with you. I did not have much time so far to revisit this api and other to add the proper comment. I've added a comment now at the top the api but it will need more clean up and explanation in the feature.